### PR TITLE
Fix basic-app benchmark application

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Fairly basic PR that just fixes the `basic-app` benchmark application. The api route was throwing an error since it wasn't returning a `Response`, and renamed some `pages/` directory routes so they become available.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: d3eac099aea8e4efdb084ab13237e4cb08f44ae1

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: bench/basic-app/app/api/app/route.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export function GET() {
-  return { name: 'John Doe' }
+  return new Response(JSON.stringify({ name: 'John Doe' }))
```
Comment (ijjk) [inline bench/basic-app/app/api/app/route.js:2]:
```suggestion
  return Response.json(({ name: 'John Doe' })
```
We probably want to use this for correct content-type header as well

File: bench/basic-app/pages/api/index.js [hunk 2]
```diff
@@ -1,3 +0,0 @@
-export default function handler(req, res) {
```
Comment (ijjk) [inline bench/basic-app/pages/api/index.js:1]: Would be good to keep these but prefix them with `page-routes` or something

File: bench/basic-app/pages/pages/index.js
Comment (Ethan-Arrowood) [inline bench/basic-app/pages/pages/index.js:1]: realizing now this is `pages/pages` so its actually available when you go to `localhost/pages`. Will restore.

File: bench/basic-app/pages/api/index.js (hunk 2 above)
Comment (Ethan-Arrowood) [reply bench/basic-app/pages/api/index.js:1]: Do the API endpoints get picked up from within the `pages/` directory if the `app/api` exists?

Comment (ijjk) [reply bench/basic-app/pages/api/index.js:1]: yeah as long as sub-path differs e.g. `pages/api/page-api.js`

Comment (Ethan-Arrowood) [reply bench/basic-app/pages/api/index.js:1]: Ah okay I must've been spelling something wrong. Works now. Restoring with a new name.

**Review by ijjk — APPROVED** [review]

**Review by leerob — APPROVED** [review]

**Pull request merged**